domains/akashr.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CruxCoder7",
+        "email": "akashrangesh07@gmail.com"
+    },
+    "record": {
+        "CNAME": "akashr.vercel.app"
```

Comment:
we dont support vercel

Reply:
Can you please tell me the hosting providers that you support?

Reply:
we support almost every platform except vercel and netlify